src/main/java/com/sarahhand/fractals/view/FractalsUI.java

```diff
@@ -147,12 +151,14 @@ public void mouseClicked(MouseEvent me) {
 				int buttonType = me.getButton();
 				int mouseX = me.getX();
 				int mouseY = me.getY();
-				if(buttonType == LEFT_CLICK) {
-					fractalConfig = createNewConfig(frameDimension, mouseX, mouseY, ZOOM_IN_FACTOR,
-							MAX_DWELL_INCREASE);
-				} else if(buttonType == RIGHT_CLICK) {
-					fractalConfig = createNewConfig(frameDimension, mouseX, mouseY, ZOOM_OUT_FACTOR,
-							MAX_DWELL_DECREASE);
+				if(mouseX < frameDimension.width && mouseY < frameDimension.height) {
```

Comment:
Do these checks with the frameDimension really ensure that zoom only happens if the user clicks inside the image?